website/oauth/models/__init__.py

```diff
@@ -443,3 +443,63 @@ def absolute_api_v2_url(self):
     # used by django and DRF
     def get_absolute_url(self):
         return self.absolute_api_v2_url
+
+class ApiOAuth2PersonalToken(StoredObject):
+    """
+    Store information about recognized OAuth2 scopes. Only scopes registered under this database model can
+        be requested by third parties.
+    """
+    _id = fields.StringField(primary=True,
+                             default=lambda: str(ObjectId()))
+
+    token_id = fields.StringField(default=lambda: uuid.uuid4().hex,  # Not *guaranteed* unique, but very unlikely
```

Comment:
Also: as far as I can tell, the `token_id` field isn't an identifier, but rather the actual value of the access token. (eg the string sent as proof that the user is allowed access)

In that case, perhaps `token_value` or such would be a better name for this field?